etc/poco/bundle/main-test.json

```diff
@@ -79,6 +79,66 @@
 			"mechanism": "group",
 			"principals": ["micros-sv--github-for-jira-dl-admins"],
 			"allowed": true
+		},
+		{
+			"name": "allow Microscope to call admin dlq endpoint",
+			"path": "/api/internal/admin/dlq",
```

Comment:
We don't have an implementation for this endpoint, do we? For the DSS services, the panels are the same but each service has its own implementation for replaying messages (i.e. for [CP](https://bitbucket.org/atlassian/dss-commit-processor/src/master/src/main/java/cloud/atlassian/fusion/dss/commitprocessor/rest/internal/admin/QueueResource.java)). From my understanding, we never really implemented that for GH and the panels are there just by default?

Reply:
I could be wrong though but worth double checking